about.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>KatherineAboutMe</title>
+    <link href="https://fonts.googleapis.com/css?family=Rock+Salt" rel="stylesheet">
+    <link href="styles/portfolio.css" rel="stylesheet">
+    <link href="styles/styles.css" rel="stylesheet">
+    <link href="styles/normalize.css" rel="normalize">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  </head>
+  <body id ="bg">
+    <header></header>
+    <nav class = "nav">
+      <section class ="column-layout">
+        <section class = "maincolumn">
+        <h3> <a href="index.html" target="_blank">Home</a></h3>
```

Comment:
Making the `target="_blank"` for the links forces each link to open in a separate tag.  Some people, myself included find this annoying.